src/lib.rs

```diff
@@ -279,12 +280,18 @@ impl GlobError {
     pub fn error(&self) -> &io::Error {
         &self.error
     }
+
+    /// Consumes self, returning the _raw_ underlying `io::Error`
+    pub fn into_error(self) -> io::Error {
```

Comment:
Its eems a bit weird to have a `into_error()` method on a type that already is a Error - maybe something like `into_io_error()` to be more explicit?

Reply:
I went with `into_error()` because the method directly above it (which returns `io::Error`) is called just `error()`. I do agree that I like `into_io_error` better, but should I also create an `io_error()` method and deprecate `error()`?

Reply:
Ah, I missed that there is precedent with the `error()` method. I guess then the name `into_error()` is fine.